stmhal/Makefile

```diff
@@ -321,6 +321,16 @@ $(BUILD)/%_$(BOARD).c $(HEADER_BUILD)/%.h $(HEADER_BUILD)/%_af_const.h $(BUILD)/
 $(BUILD)/pins_$(BOARD).o: $(BUILD)/pins_$(BOARD).c
 	$(call compile_c)
 
+GEN_STMCONST_HDR = $(BUILD)/modstm_const.gen.h
+GEN_STMCONST_QSTR = $(BUILD)/modstm_qstr.h
+CMSIS_MCU_LOWER = $(shell $(SED) -n -e 's/\#define[[:space:]]*\(STM32F...x.\)/\1/p' $(wildcard boards/$(BOARD)/stm32f*_hal_conf.h) | tr '[:upper:]' '[:lower:]')
```

Comment:
Do we really need such magic?  Can we just force each port to define the cmcis header?  (in mpconfigboard.mk)